Source/Blazorise/Base/BaseInputComponent.razor.cs

```diff
@@ -125,6 +149,20 @@ protected virtual void ReleaseResources()
         }
     }
 
+    /// <summary>
+    /// Initializes parameters that are not going to be changed again.
+    /// </summary>
+    protected void IntializeParameters()
```

Comment:
Did you end up not using this method? Can you check?

Reply:
No, I left it on purpose. I plan to use it more in the future. It will be part of the refactoring `Value`.

Reply:
You were right. It was a duplicate. Thanks.